api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -52,12 +53,12 @@ public CohortDefinition getAllPatientsWhoAreActiveAndStable() {
     cd.addSearch(
         "2",
         EptsReportUtils.map(
-            ageCohortQueries.createXtoYAgeCohort("moreThanOrEqual2Years", 2, 200),
+            ageCohortQueries.createXtoYAgeCohort("moreThanOrEqual0Years", 0, 200),
```

Comment:
This change affects D1 which is not part of the ticket, it should be reverted.

Reply:
moved back